lncfg/routing.go

```diff
@@ -3,4 +3,6 @@ package lncfg
 // Routing holds the configuration options for routing.
 type Routing struct {
 	AssumeChannelValid bool `long:"assumechanvalid" description:"Skip checking channel spentness during graph validation. This speedup comes at the risk of using an unvalidated view of the network for routing. (default: false)"`
+
+	StrictZombiePruning bool `long:"strictgraphpruning" description:"If true, then the graph will be pruned more aggressively for zombies. In practice this means that edges with a single stale edge will be considered a zombie."`
```

Comment:
Should note that this is enabled by default for neutrino. Any reason to not have it apply for all backends though?

Reply:
i think the rationale is that for full nodes are more likely to be serving some sort of graph indexing/metrics we want to make it opt-in to not disrupt their services. that said, i suppose it doesn't preclude the possibility of someone having a neutrino backed metrics node..

Reply:
Yeah also we don't want to cause a "shock" to those that track things like the number of nodes without really understanding how we track/prune them. Remember back when we first implemented pruning and people thought that someone "DoS'd" 20% of the nodes off the network 😅